### DISCUSSION TITLE
Is there a way to disable groupByYear in taxonomy listing?

### DISCUSSION THREAD

**Discussion by vinceimbat** [jpanther/congo#124] [Q&A]
In the blog listing, when I disable groupByYear, it gets disabled. In taxonomy listing, this doesn't seem to work for me. Is this just how the theme is currently designed? What can I possibly do to disable groupByYear in taxonomy listing?

**Comment by jpanther** [comment]
This is not currently possible, however I can look to add this to a future update. In the meantime you can change this functionality by overriding the term template at `layouts/_default/term.html`. If you copy the theme template into your local site and remove the `<h2>` that outputs the `{{ .Key }}`, and the `<hr>` from your local copy you will achieve a result without the year groupings.

**Reply by vinceimbat** [reply]
Thanks @jpanther ! This did work. Looking forward to your future update on this :)
